apps/web/src/ee/translations/components/EditGroupSidebar.tsx

```diff
@@ -115,7 +115,7 @@ export const EditGroupSidebar = ({
       }
     >
       <FormProvider {...methods}>
-        <GroupFormCommonFields edit={true} control={control} readonly={readonly} />
+        <GroupFormCommonFields edit={true} control={control as Control<ICreateGroup>} readonly={readonly} />
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** I'm not sure why this appeared now, but my build kept failing locally without this.‏